selenium/src/main/java/com/deque/html/axecore/selenium/AxeBuilder.java

```diff
@@ -739,10 +744,25 @@ private Results analyzePost43x(final WebDriver webDriver, final Object rawContex
 
     String prevWindow = WebDriverExtensions.openAboutBlank(webDriver);
     injectAxe(webDriver);
+    String partialResString = "";
+    try {
+      partialResString = objectMapper.writeValueAsString(partialResults);
+    } catch (Exception e) {
+      throw new RuntimeException(e);
+    }
+    int sizeLimit = 15_000_000;

```

Comment:
We should be consistent with the `sizeLimit` we've set with the integrations [core-npm has it set at `60_000_000`](https://github.com/dequelabs/axe-core-npm/blob/develop/packages/playwright/src/index.ts#L290)

```suggestion
    int sizeLimit = 60_000_000;
```